features/tree_metric_configuration/edition.feature

```diff
@@ -9,9 +9,9 @@ Feature: Metric Configuration edition
     And I am signed in
     And I have a sample configuration
     And I have a sample reading group
-    And I have a sample metric configuration within the given mezuro configuration
+    And I have a sample tree metric configuration within the given mezuro configuration
     When I am at the Sample Configuration page
-    Then I should not see Edit within table
+    Then I should not see Edit within table#tree_metric_configurations
```

Comment:
It really does work! Actually, without this selector, the step would find 2 tables and would be unable to continue